backend/components/contacts/src/main/java/co/airy/core/contacts/Stores.java

```diff
@@ -158,6 +161,11 @@ public void onApplicationEvent(ApplicationReadyEvent applicationReadyEvent) {
                 })
                 .to(applicationCommunicationContacts);
 
+        // Conversations data necessary for the contact messages endpoint
+        builder.table((new ApplicationCommunicationConversations().name()), Materialized.as(conversationsStore));
```

Comment:
Rather than replicate the entire conversations infrastructure in this app I decided to turn the `communication` app into a producer so that other apps can use its output for their own conversation endpoints.

However, this is an exploratory feature because it introduces an additional layer of dependency between the app and the application topics.